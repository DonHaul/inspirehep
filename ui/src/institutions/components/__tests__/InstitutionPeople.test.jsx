import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import InstitutionPeople from '../InstitutionPeople';

describe('InstitutionPeople', () => {
  it('renders', () => {
    const recordId = 123;
    const { asFragment } = render(
      <MemoryRouter>
        <InstitutionPeople recordId={recordId} />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
