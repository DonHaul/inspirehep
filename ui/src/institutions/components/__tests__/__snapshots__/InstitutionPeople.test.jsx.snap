// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstitutionPeople renders 1`] = `
<DocumentFragment>
  <span>
    List of 
    <a
      href="/authors?q=positions.record.$ref:123"
    >
      People
    </a>
  </span>
</DocumentFragment>
`;
